CHANGELOG.md

```diff
@@ -1,35 +1,61 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [ 0.5.3 ] - [ 2023-01-02 ]
```

Comment:
Apparently to pass the "release" version of the CI you need your branch name to start with "release-"
This will run the assets.yml workflows on top of the test.yml.
But in fact it's independent from actually releasing, so it's fine for this time (as long as the assets workflows don't fail, which you don't know in advance)